Dockerfile

```
@@ -1,7 +1,7 @@
-ARG JAVA_VERSION=16
+ARG JAVA_VERSION=17
 ARG JVM_FLAVOR=hotspot
 
-FROM adoptopenjdk:${JAVA_VERSION}-jdk-${JVM_FLAVOR} AS builder
+FROM openjdk:${JAVA_VERSION}-slim AS builder
```

You likely want the JDK for compiling the project.

`openjdk:17-jdk-slim`


```suggestion
FROM openjdk:${JAVA_VERSION}-jdk-slim AS builder
```